market-service/src/main/java/rs/edu/raf/banka1/mapper/FutureMapper.java

```diff
@@ -0,0 +1,68 @@
+package rs.edu.raf.banka1.mapper;
+
+import org.springframework.stereotype.Component;
+import rs.edu.raf.banka1.model.ListingForex;
+import rs.edu.raf.banka1.model.ListingFuture;
+import rs.edu.raf.banka1.model.ListingStock;
+import rs.edu.raf.banka1.model.OptionsModel;
+import rs.edu.raf.banka1.model.dtos.ListingBaseDto;
+import rs.edu.raf.banka1.model.dtos.ListingForexDto;
+import rs.edu.raf.banka1.model.dtos.ListingFutureDto;
+import rs.edu.raf.banka1.model.dtos.OptionsDto;
+@Component
+public class FutureMapper {
+    public void updateFuture(ListingFuture oldFuture, ListingFuture newFuture) {
```

Comment:
I think that it's better to move functions which are not related to mapper functionality (transforming DTOs to Model and Model to DTO) to a separate file.

Reply:
made it related to mapper by returning an object